code/modules/mob/living/carbon/human/species_types/golems.dm

```diff
@@ -516,7 +516,7 @@
 	attack_verb = "honk"
 	attack_sound = 'sound/items/airhorn2.ogg'
 	prefix = "Bananium"
-	special_names = null
+	special_names = list("Peel","Cream Pie","Split","Honk")

```

Comment:
Bananium golems override the `random_name` proc, so these names aren't used.

Reply:
Thanks, I missed that one.

Reply:
```DM
/datum/species/golem/bananium/random_name(gender,unique,lastname)
	var/clown_name = (prob(50)? pick(special_names) : pick(GLOB.clown_names))
	var/golem_name = "[uppertext(clown_name)]"
	return golem_name
```

????